python/lsst/ts/atmcssimulator/schemas/tel_azEl_mountMotorEncoders.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "id": {
+            "enum": [
+                "azEl_mountMotorEncoders"
+            ]
+        },
```

Comment:
The data type of "id" is "string" instead of "enum". You would get the `"id": "azEl_mountMotorEncoders"`.

Reply:
In this and all other cases with just one option I changed it to the `const` type so it can only be that value and nothing else.